### ISSUE TITLE
Widget "preseeds" wrong value into price field

### ISSUE THREAD

**Issue by MaxRink** [pretix/pretix#1279]
The Pretix Widget seeds the wrong value into the price filed if you enable variable pricing.

````html
<input type="number" placeholder="0" min="45,00" name="price_6" step="any" class="pretix-widget-pricebox-price-input">
````

is the issue, where placehoulder should not be 0.

This can be observed at chaos-party-ulm.de

**Comment by pc-coholic** [comment]
I tried to recreate this with the latest pretix 2.7.0 release and was not able to reproduce this issue...

Would you mind checking which version of pretix you are running?

**Comment by MaxRink** [comment]
Im running 2.7.0 in the official docker container

**Comment by raphaelm** [comment]
The error only occured in German locale (a decimal comma versus decimal point thing) and will be fixed in 2.7.1. Thanks for reporting!
